src/persist/src/persist.proto

```diff
@@ -55,7 +55,7 @@ message ProtoUnsealedBatchMeta {
 }
 
```

Comment:
the desc field of ProtoTraceBatchInline isn't really correct anymore. we could document that it means the associated data is part (but maybe not the entirety) of the batch, but maybe we should just remove the field instead? it was just a nice to have anyway. too bad we won't know ahead of time how many chunks there will be, it'd be nice to store "this is chunk N of M" in the inline metadata. the N part at least is easy